spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/util/ObservableReactiveUtil.java

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
I would put this in a separate module, something like `spring-cloud-gcp-reactive-utils` because as we build out more reactive functionality, these converters will get more useful.

Reply:
If it's not used anywhere else right now, I would leave it for the time being.